app/components/exercises/AddExercise.js

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import { Link } from 'react-router';
+import { StepsTextField, StepsSelectField, StepsMenuItem, StepsRaisedButton } from '../material-style';
+
+export default ({ handleChange, handleSubmit, open, errors}) => {
+  return (
+    <div id="add-exercise">
+    {
+      !open ?
+        <StepsRaisedButton
+          label="Add New Exercise"
+          onClick={ () => handleChange('open', true) }
+          backgroundColor="#005B96"
+        /> :
+        <form onSubmit={ handleSubmit }>
+          <fieldset className="form-box">
+          <legend style={{width: '145px'}}>New Exercise</legend>
+            <div className="row">
+              <div className="col-xs-12">
+                <StepsTextField
+                  floatingLabelText="Title"
+                  errorText={ errors.title }
+                  fullWidth={ true }
+                  onChange={(evt) => handleChange("title", evt.target.value) }
+                />
+              </div>
+            </div>
+            <div className="row">
```

Comment:
This use of `<row><col><field></col></row>` seems redundant if you are just using all 12 columns, especially if you are using the `fullWidth` attribute. Im guessing you could remove them and have the same styling effect